broker/mosquitto.conf

```diff
@@ -9,7 +9,7 @@
 # given, the default listener will not be bound to a specific
 # address and so will be accessible to all network interfaces.
 # bind_address ip-address/host name
-bind_address localhost
```

Comment:
bind_address should probably remain localhost, 
then make a copy that you .gitignore with the local-ip